client/src/pages/Profile.js

```diff
@@ -1,19 +1,21 @@
-import React from 'react';
+import React, { useState } from 'react';
 import styled from '@emotion/styled';
 import Header from '../components/Header';
 import FullContainer from '../components/FullContainer';
-import ProfileInfo from '../components/ProfileInfo';
-import example from '../assets/profile_example.jpg';
 import ProfileImage from '../components/ProfileImage';
+import PlayerDropdown from '../components/PlayerDropdown';
+import FooterButton from '../components/FooterButton';
+import example from '../assets/profile_example.jpg';
 
 const ImageBlur = styled.div`
-  background-image: url(${example});
+  background-image: url(${(props) => props.src});
   background-size: cover;
+  background-position: center;
   background-repeat: no-repeat;
   height: 40%;
   width: 100vw;
```

Comment:
and this is the second culprit for the scroll-bars I get in non mobile view on win10/Chrome 🏹